src/server/api-server.js

```diff
@@ -77,7 +78,7 @@ class APIServer extends EventEmitter {
   startListening() {
     return new Promise((resolve, reject) => {
       this._expressServer = this._expressApp.listen(
-        this._port, this._host, () => {
+        this._port, this._serveHost, () => {
```

Comment:
It's been a while since I tested it but IIRC '0.0.0.0' was accepted but it had no effect when the API was running not on localhost but on the IP address of the docker container.